.circleci/config.yml

```diff
@@ -5,7 +5,12 @@ jobs:
       - image: 'redislabsmodules/rmbuilder:latest'
     steps:
       - checkout
-      
+      - run:
+          name: Fetch submodules
+          command: >-
+            git submodule init &&
```

Comment:
git submodule update --init --recursive
# instead of 'git submodule init; git submodule update'